distributed/dashboard/components/scheduler.py

```diff
@@ -276,25 +276,30 @@ class MemoryColor:
     orange: float
     red: float
 
-    def __init__(self):
+    def __init__(self, neutral_color="blue", target_color="orange", terminated_color="red"):
+        self.neutral_color = neutral_color
+        self.target_color = target_color
+        self.terminated_color = terminated_color
```

Comment:
Worth noting that we could also configure the colors for the worker memory bars when they are approaching the memory limit (i.e. possibly spilling) or have been terminated; right now I've only overridden the neutral color for the GPU plot but not sure if it makes sense to use different colors for these other states?